tests/python/frontend/darknet/test_forward.py

```diff
@@ -23,16 +24,16 @@
 """
 import numpy as np
 import tvm
-from tvm import te
 from tvm.contrib import graph_executor
 from tvm.contrib.download import download_testdata
 
-download_testdata.__test__ = False
 from tvm.relay.testing.darknet import LAYERTYPE
 from tvm.relay.testing.darknet import __darknetffi__
 from tvm.relay.frontend.darknet import ACTIVATION
 from tvm import relay
 
+download_testdata.__test__ = False
```

Comment:
Not sure, can i remove？

Reply:
yeah i believe so, i tried it locally and it seems to be ok for this particular test. it seems like that's designed to tell pytest that download_testdata isn't a test, but i'm not sure why it would think that to begin with.

Reply:
Ok！